features/string-raw.yml

```diff
@@ -0,0 +1,6 @@
+name: String raw()
+description: The `normalize()` static method is a tag function of template literals.
```

Comment:
```suggestion
description: The `String.raw()` static method interpolates template literal substitutions, but ignores escape sequences. It is the tag function for template literals.
```

Reply:
This one is hard to write about. Florian, if you have a take on this, I'd like to hear it.